hyper/client/aws/ec2.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/base64"
 	"fmt"
+	hyperdriveTypes "github.com/gohypergiant/hyperdrive/hyper/types"
```

Comment:
Don't have a ton of context on this file. Is there somethign specific to look for?

Reply:
this was part of a refactor, I moved the type definitions into their own folder both so they would be easier to find and so that they could be imported in multiple places without having cyclical imports